docs/content/docs/build/packages.md

```diff
@@ -1,14 +1,72 @@
 ---
 title: Packages
 menuPosition: 7
-editor: tylerbutler
-status: unwritten
+author: tylerbutler
 ---
 
-Explain what the fluid-framework package is.
-Explain the namespaces (scopes).
-List of all the packages in fluid-framework.
+The Fluid Framework is a multi-layered system consisting of dozens of individual npm packages. Most developers will only
+need two or three of these packages for typical Fluid development: The `fluid-framework` package, which contains the
+public API for the Fluid Framework, and a service-specific client package, such as the
+`@fluidframework/tinylicious-client` package.
 
-Experimental scope and packages.
+## Primary API: fluid-framework
 
-List other scopes and their meaning (or link to the wiki).
+The `fluid-framework` package bundles a collection of Fluid Framework client packages for easy use when paired with a
+corresponding service client package, such as the `@fluidframework/azure-client` package. The `fluid-framework` package
```

Comment:
i thought it was a bit odd that this sentence was basically the same as the previous one, but used a different example.  may be better to use the same example and link to the available service clients page

Reply:
Agreed. I think we can just axe the first sentence. GitHub is being odd about suggestions atm so I can't select it.